### ISSUE TITLE
New Logo

### ISSUE THREAD

**Issue by aneagoie** [zeroDevs/coding_challenge-13#63]
A new cleaner logo for the website will be a great PR. We can also add this to the favicon as well.

**Comment by The24thDS** [comment]
How about something like this?
![annotation 2019-01-09 214327](https://user-images.githubusercontent.com/26633429/50924320-e9478700-1457-11e9-83ee-e953fd1e0567.jpg)

**Comment by mortenkos** [comment]
What about this one? Colors are changeable. Please no pity and be honest :D 
![fullstacktrendslogo](https://user-images.githubusercontent.com/25040343/50980713-09cc1b80-1502-11e9-8a77-46580c9c46bf.png)

**Comment by aneagoie** [comment]
I think we should keep the color monotone (one color) to make it work visually. Maybe just a logo without the words (words can be on the side)

**Comment by The24thDS** [comment]
This is what I have come up with.
Before scrolling the page:
![logo1](https://user-images.githubusercontent.com/26633429/50989270-9208eb80-1517-11e9-8927-8b539ad74c84.jpg)
When the page is scrolled:
![logo2](https://user-images.githubusercontent.com/26633429/50989327-b5cc3180-1517-11e9-89eb-8162cdffbb7b.jpg)

**Comment by The24thDS** [comment]
And this is the logo in the footer
![logo footer](https://user-images.githubusercontent.com/26633429/51030889-73e8cd00-15a3-11e9-86ce-53be3603bdf1.jpg)

**Comment by aneagoie** [comment]
I think the coloring should match the theme of the website. Green, just doesn't seem right don't you think?

**Comment by The24thDS** [comment]
I took that color from palleton but I have to agree that it doesn't really fit. I changed the color to the ones of the buttons. What do you think about it now?
![logo](https://user-images.githubusercontent.com/26633429/51089122-c9bea000-1770-11e9-93e8-0e184a5cb0ff.jpg)

**Comment by aneagoie** [comment]
Thoughts on this @MattCSmith ?

**Comment by MattCSmith** [comment]
I like the icon, although If the text is to be part of the image, it could benefit from some styling.
Otherwise the text could be styled via CSS, which might be a better approach. 

Im not sold on the color, as I feel it blends in a little. Perhaps the following may help:
- Arrow and brackets could be white (maybe with some transparency)
- Try adding a border around the stack (again maybe a slightly transparent white)
- Same as the previous point, but with a small gap between the stack and the border

Excuse my very poor paint mockup, but something like this, maybe:
![image](https://user-images.githubusercontent.com/6190356/51242546-2344eb80-1978-11e9-998f-4962520c5911.png)

**Comment by The24thDS** [comment]
I got something like that. Also, the text is not part of the images.
![logo1](https://user-images.githubusercontent.com/26633429/51244205-21d1ee00-198d-11e9-8308-259fcf6d116f.jpg)

**Comment by MattCSmith** [comment]
I feel that works a little better, although if I'm honest I still feel it blends a little to much. 
I wonder if either:
- The border should be thicker or less transparent or both
- We should find a different color, that still matches the theme

Another option might be to make the nav bar permanently white (eg. No on-scroll). What does it look like when with the scrolled navbar?

**Comment by The24thDS** [comment]
I have come up with this solution @MattCSmith, I think it looks better
(This is with a logo-size of 65)
![image](https://user-images.githubusercontent.com/26633429/51257047-71c1ac80-19af-11e9-8e9e-e303259dc798.png)
This is how it looks when the page is scrolled: (I have also changed the size from 40 to 50 here)
![image](https://user-images.githubusercontent.com/26633429/51257232-d67d0700-19af-11e9-811e-6316983c660c.png)
Footer:
![image](https://user-images.githubusercontent.com/26633429/51257272-ec8ac780-19af-11e9-914c-43dad28f397a.png)

**Comment by MattCSmith** [comment]
I think that looks a lot better. What do you think @aneagoie ?

**Comment by aneagoie** [comment]
Looks much better. Could we maybe play around with the font to make it look more appealing? What do you all think?

**Comment by MattCSmith** [comment]
@aneagoie I thought the same thing earlier in the issue. The text wont be apart of the logo, from what I understand. This will be added and styled in the HTML/CSS, I think?

**Comment by The24thDS** [comment]
Yes. The text is not an image. It can be styled using CSS, no problem.

Pe 17 ian. 2019, la 17:12, Matt Smith <notifications@github.com<mailto:notifications@github.com>> a scris:


@aneagoie<https://github.com/aneagoie> I thought the same thing earlier in the issue. The text wont be apart of the logo, from what I understand. This will be added and styled in the HTML/CSS, I think?

**Comment by rahmot** [comment]
Hi guys, check this out.
[content://media/external/file/94046](url)

**Comment by marcosandri-dev** [comment]
I would remove another laptop from the logo. If not all of them; the < arrow > thing is already very powerful, I read it as "trends in web development" and the important thing I think is "trends", not the full-stack which is anyway explained in the product name!
Like this is feels... bloated, and I don't know much about web design and marketing, but I know that what I feel is always very important, and I feel is like "too much", somehow! :)

**Comment by ChristinaRepou** [comment]
What about something like this? Thank you!
![logo](https://user-images.githubusercontent.com/42965573/51404988-b428ec00-1b5d-11e9-872e-ab002e35ba00.png)

**Comment by rahmot** [comment]
Hi guys, please check this out
![full stack3](https://user-images.githubusercontent.com/39836912/51405073-99527980-1b55-11e9-9035-5bf196d867e7.jpg)

**Comment by ChristinaRepou** [comment]
What about something like this?
![logo](https://user-images.githubusercontent.com/42965573/51405357-ceaf9500-1b5e-11e9-88e3-1f0f9b73229e.png)

**Comment by The24thDS** [comment]
@ChristinaRepou That actually looks cool but I don't know how much of what is on the screen will be visible at the dimensions of the logo.

**Comment by rahmot** [comment]
Please check out this concept.
![full stack4](https://user-images.githubusercontent.com/39836912/51409005-db34ed00-1b60-11e9-9c9d-de76bf55f6f3.jpg)

**Comment by ChristinaRepou** [comment]
There is also this design!
![fullstack trends logo](https://user-images.githubusercontent.com/42965573/51412555-987d1000-1b74-11e9-8844-880673e2bdf9.png)

**Comment by aneagoie** [comment]
Some great examples here. Can everyone give a 👍 to the one you like best?

**Comment by MattCSmith** [comment]
Wow this issue exploded this evening and they are all pretty awesome.

I do agree though that some may suffer from scaling to the actual size of the logo/favicon. But some tweaks could be made to accommodate that perhaps.

@ChristinaRepou @rahmot @The24thDS Could you confirm the origins of the designs. Ie are these completely hand-made, created by a logo creator or heavily inspired by another logo. Since this will be a live site, we will need to ensure that we are compliant with copyright conditions of anything we use within the site.

**Comment by The24thDS** [comment]
@MattCSmith Of course. Fully made by me using Adobe Illustrator. But a laptop is still a laptop so you can't design it very differently 🤣

**Comment by ChristinaRepou** [comment]
Also mine have been made by me with another programme (canva).

**Comment by ChristinaRepou** [comment]
@MattCSmith there is not a problem with my designs too although I made them with canva.

**Comment by marcosandri-dev** [comment]
I like @The24thDS logo without the other laptops actually but I cannot vote it since it's not in the list :P

**Comment by The24thDS** [comment]
One laptop logo:
![image](https://user-images.githubusercontent.com/26633429/51431315-d854fd80-1c2f-11e9-9cac-ceca2ceec22b.png)
![image](https://user-images.githubusercontent.com/26633429/51431317-e3a82900-1c2f-11e9-88d3-e5343b6a8d20.png)
![image](https://user-images.githubusercontent.com/26633429/51431320-eacf3700-1c2f-11e9-99a8-346295281341.png)
In my opinion it doesn't look that good.

**Comment by The24thDS** [comment]
Two laptops logo:
![image](https://user-images.githubusercontent.com/26633429/51431328-05a1ab80-1c30-11e9-92a9-771a26052f9b.png)
![image](https://user-images.githubusercontent.com/26633429/51431331-0a665f80-1c30-11e9-8cdb-350c856e2d91.png)
![image](https://user-images.githubusercontent.com/26633429/51431336-13573100-1c30-11e9-9276-add6fadfef18.png)

**Comment by MattCSmith** [comment]
Hi @rahmot, No problem. 

How ever I do not think we will be able to use the triangular one. As cool as it is, its listed as a logo created by a logo maker website?

![image](https://user-images.githubusercontent.com/6190356/51470838-efe5d080-1d6c-11e9-922e-aa76e3a0d9f0.png)

**Comment by aneagoie** [comment]
@rahmot HI! Would you be able to create a logo like the above from scratch so we can us it?

**Comment by rahmot** [comment]
Hi @aneagoie which of the logo are you referring to because there are so many logos above. Thanks

**Comment by MattCSmith** [comment]
@rahmot My concern here, is that if we used that logo (the triangular one)  we would either need to purchase the license or credit them. 
Apart from a change in colors and fonts/text the logo's are identical. If we proceed without doing either purchasing the license or crediting them, we could face repercussions.

**Comment by rahmot** [comment]
@MattCSmith I am not insisting on us using the triangular logo, a new logo can be designed. Guys check this out
![full stack65](https://user-images.githubusercontent.com/39836912/51483377-37825180-1d99-11e9-9693-e38a17654903.jpg)

**Comment by marcosandri-dev** [comment]
@The24thDS you're right, 2 laptops is better!

**Comment by rahmot** [comment]
![full stack45](https://user-images.githubusercontent.com/39836912/51487675-e9268000-1da3-11e9-8d50-c9f1ddf3c90f.jpg)


![full stack567](https://user-images.githubusercontent.com/39836912/51487550-be3c2c00-1da3-11e9-86f3-9defa51d1b4f.jpg)

![full stack43](https://user-images.githubusercontent.com/39836912/51487743-13783d80-1da4-11e9-9643-63cf0f46144d.jpg)

**Comment by aneagoie** [comment]
Some really great ideas here. Can we all vote on the ones that we like by the end of day today? It would be nice to have this implemented. If we can't come up with a favourite in the next 2 days, @MattCSmith and I will just pick one, but it would be nice if the community got to decide.

**Comment by mortenkos** [comment]
My vote goes for the Rahmot's last logo

**Comment by aneagoie** [comment]
Ok, it looks like we have a clear winner based on the votes on Discord. @rahmot do you want to go ahead and add the logo, or would you like to pass this on to somebody else? (including updating the favicon)

**Comment by rahmot** [comment]
@aneagoie I will be glad to go ahead with the logo. Thanks

**Comment by rahmot** [comment]
Please, someone should help me out. How will I add it? Thanks

**Comment by rahmot** [comment]
Good day @MattCSmith please, I don't understand what fork and clone the repo is all about. I'm a complete beginner in web development. I tried to Google search but the explanation wasn't clear. Please break it down and I will the instructions obediently. Thanks
